app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def user_data
+    @user_data = User.select('name, COUNT(posts.id) AS post_count')
+      .joins(:posts)
+      .group('users.id')
+  end
```

Comment:
-[ ] For proper and best practices, kindly name the method that will render the index page as `index` so it goes with your view files 👍 

### VIEW:
![image](https://github.com/Samuelwanza/sema/assets/50990925/65cde0f4-7b30-4b8c-8b73-224fd80963ac)